terraform/aws/approzium.config.yml

```diff
@@ -1,13 +1,15 @@
-# This config file will be used by the deployed Approzium authenticator.
-# Remember to uncomment variables and change them to suit your setup.
-# host: 127.0.0.0
-# httpport: 6000
-# grpcport: 6001
-# loglevel: info
-# logformat: text
-# lograw: false
-# secretsmanager: local
-# or .. if using Vault, use:
-# secretsmanager: vault
-# vaulttoken: ...
-# vaultaddr: ...
+# Please edit it appropriately before production deploys.
+# See https://approzium.com/docs/configuration for more.
+---
+listener:
+  grpc_port: 6001
+  host: "127.0.0.1"
+  http_port: 6000
+logging:
+  log_format: json
+  log_level: info
+  log_raw: false
+secrets:
+  secrets_manager: "local"
+tls:
+  disable_tls: true
```

Comment:
All of this updates the sample config to the current one we'd expect in version 0.2.0.